docs/contributtion/experimental-code.md

```diff
@@ -0,0 +1,167 @@
+
+### Feature flag naming convention
+
+All feature flags follow a prefix naming convention that indicate status.
+
+#### Flags prefixed with `enable-*`
+
+- Contain new features that we'd like consuming projects to test
+- Are generally stable and unlikely to change but may change based on user
+  feedback
+- May require some manual migration or code changes within your project
+- Are documented in the main package README file
+- Need user feedback to ensure we've met all concerns relating to this feature
+
+If you use these flags, make sure to check our release notes where we'll outline
+any changes to them across our releases.
+
+#### Flags prefixed with `enable-v#-*`
+
+As usage of an existing flag increases or we determine a feature to be of high
+importance, we'll "commit" it to a future major release and rename it to use the
+`enable-v#-*` prefix. At this point the API or functionality behind this flag is
+now fixed and won't change. We intend to ship this flag as "on by default" in
+the major version indicated in the name. e.g. `enable-v3-some-feature`
+
+All breaking changes will be shipped as `enable-v3-*` flags within the current
+major release (v3). This enables projects to opt-in to breaking changes earlier
+and at their own pace avoiding one huge changeset when upgrading to the next
+major release. In theory, if all `enable-v3-*` flags are enabled within your
+project before the v3 release, no changes should need to be made when updating
+to v3.
+
+For a flag to be committed to a release and renamed to `enable-v#-*` it must:
+
+- Be tested with early adopters
+- Be fully covered in tests (Unit, Visual)
+- Be documented in storybook
+- Be documented on https://spirit.design
+- Have an automated migration script (codemod) available, where possible
+
+### Documenting Feature Flags
+
+Within the experimental section of the `@lmc-eu/spirit-web-twig` Storybook, there is a
+"Feature Flags" folder. This contains stories and written documentation covering
+every available flag and how to configure them. Folders and stories within this
+section in storybook show components with all feature flags turned on.
+
+All the currently available feature flags, as well as how to enable them, is
+documented in the main package README file.
+
+## Expanding existing APIs
+
+New experimental functionality can sometimes be added without the need of an
+`UNSTABLE_` export or a feature flag. Oftentimes this ends up being a new prop
+on a component. The documentation associated with this prop (TypeScript types, Storybook controls, etc.) will state that it's experimental.
+
+## Moving to stable
+
+Over time it becomes apparent an experimental API has stabilized and suits the
+needs of most users. If it isn't a breaking change, and there hasn't been much
+movement, it can be moved from "experimental" status to be "stable".
+
+To initiate a move to stable, open a new issue to request the team to evaluate
```

Comment:
we really want someone to open a new issue? :)

Reply:
Copy changed.